docs/index.md

```diff
@@ -85,6 +85,7 @@ provider "tanzu-mission-control" {
     username      = var.username           # optionally use TMC_SM_USERNAME env var
     password      = var.password           # optionally use TMC_SM_PASSWORD env var
   }
+  ca_file = var.ca_file                    # Path to Host's root ca set. The certificates issued by the issuer should be trusted by the host accessing TMC Self-Managed via TMC terraform provider.
```

Comment:
Yeah right, so this option is also there for Saas but not a mandatory field since the provider behind the scene is handling this case by setting the root ca. If the proxy is used then yes this filed comes into picture.

 In case of TMC SM since this becomes a mandatory field, we are documenting it explicitly in the usage example.